go/vt/wrangler/testlib/external_reparent_test.go

```diff
@@ -100,29 +100,24 @@ func TestTabletExternallyReparentedBasic(t *testing.T) {
 	}
 
 	// We have to wait for shard sync to do its magic in the background
-	timer := time.NewTimer(10 * time.Second)
-	defer timer.Stop()
-
-loop:
+	startTime := time.Now()
 	for {
-		select {
-		case <-timer.C:
-			// we timed out
+		if time.Since(startTime) > 10*time.Second /* timeout */ {
 			tablet, err = ts.GetTablet(ctx, oldMaster.Tablet.Alias)
 			if err != nil {
 				t.Fatalf("GetTablet(%v) failed: %v", oldMaster.Tablet.Alias, err)
 			}
 			t.Fatalf("old master (%v) should be replica but is: %v", topoproto.TabletAliasString(oldMaster.Tablet.Alias), tablet.Type)
-		default:
+		} else {
```

Comment:
This `else` and the next one are not really needed since their corresponding `if` blocks always break/terminate. I personally like the style of keeping such blocks unindented when possible.